formal-spec/measurements.tex

```diff
@@ -262,7 +262,7 @@ \subsection{Processing Time and Memory Consumption}
   \label{fig:eltime-vs-participants}
 \end{figure}
 
-In Figure \ref{fig:eltime-vs-participants}, we see that the elapsed 
+In Figure \ref{fig:eltime-vs-participants}, we see that the processing 
```

Comment:
I would also add a comment on whether this memory consumption seems reasonable. I'd say it does:

```
Haskell maps use 6 words (a word takes 8 bytes) per node. In general there are twice as many nodes as values. Each value requires two words. And storing a hash requires a word for the tag, plus the size of the hash. Then, for storing p hashes we need:

  p * 8 * (6 + 6 +  2 + 1 + 4)

So if p = 1M we need 152 MB to store the hashes, so the resulting 195 MB used to tally the votes of 1M  participants is within the memory usage range we'd expect.
```

Should we add something like that?

Reply:
Sure why not.